server/schedule/filter/filters.go

```diff
@@ -34,7 +34,8 @@ import (
 func SelectSourceStores(stores []*core.StoreInfo, filters []Filter, opt *config.PersistOptions) []*core.StoreInfo {
 	return filterStoresBy(stores, func(s *core.StoreInfo) bool {
 		return slice.AllOf(filters, func(i int) bool {
-			if !filters[i].Source(opt, s).IsOK() {
+			status := filters[i].Source(opt, s)
+			if !status.IsOK() && status != statusStoreTombstone {
```

Comment:
Maybe we can add 'not tombstone' to `status.IsOK()`?

Reply:
I think it belongs to the different types. The previous change has a problem, I have updated it.